src/components/NavBar.js

```diff
@@ -15,13 +15,6 @@ const NavBar = props => {
       <div>
         <AppBar position="static">
           <Toolbar>
-            {/* <NavLink exact to="/">
-              <img 
-                className="Header-lambda-logo"
-                src={redLambdaLogo}
-                alt="red lambda logo"
-              />
-            </NavLink> */}
```

Comment:
Do we need this commented block?

Reply:
nope!
I'm a hoarder and I like to hang on to the old stuff in case the new stuff breaks..

Reply:
Lol, trust your code and problem solving skills, it's gonna work fine.
Also, if we needed to refer to our old code we could just go back through Git and find it 💯